canonical/ubuntu.com
# feat: Add new sections to '/containers/chiselled'

## Done

- Add new sections to '/containers/chiselled' based on [copydoc](https://docs.google.com/document/d/1ZkQ-UFTwsPkMnQn8IEEjZOXMy40tkgBscw_-qq9K3nc/edit) and [design](https://www.figma.com/design/YfNc8KmYqa2e7JgbFo6lXa/24.04-U.com---%2Fcontainers?node-id=440-3441&t=Snw41ddNHSYNRMvI-0)

## QA

- Go to https://ubuntu-com-13991.demos.haus/containers/chiselled
- Check content matches the [copydoc](https://docs.google.com/document/d/1ZkQ-UFTwsPkMnQn8IEEjZOXMy40tkgBscw_-qq9K3nc/edit)
- Check design matches the [design](https://www.figma.com/design/YfNc8KmYqa2e7JgbFo6lXa/24.04-U.com---%2Fcontainers?node-id=440-3441&t=Snw41ddNHSYNRMvI-0)

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-12561
